se/formatting.py

```diff
@@ -208,7 +208,7 @@ def _get_syllable_count(word: str) -> int:
 	# 8) add one if "y" is surrounded by non-vowels and is not in the last word.
 	for i, j in enumerate(word):
 		if j == "y":
-			if (i != 0) and (i != len(word) - 1):
+			if i not in (0, len(word) - 1):
```

Comment:
Here we're comparing against integers, not sets, so I think this change is less clear.

Reply:
I have to say that I hesitated with that one slightly. According to the docs, checking membership in a tuple is "faster and less verbose" than checking against multiple values. So it does seem like a valid change. The specific pylint flag is `consider-using-in` (R1714) if you want to google it.

All that said, I will back it out. I don't have a strong opinion either way.